src/server/auth/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Express } from 'express';
```

Comment:
Why Express instead of Koa?

Reply:
Easier

Reply:
Do we still need Koa then? If not, we can remove from package.json, regen package-lock.

Reply:
Acknowledged.